src/textual/message_pump.py

```diff
@@ -78,6 +85,58 @@ def __init__(self, parent: MessagePump | None = None) -> None:
         self._mounted_event = asyncio.Event()
         self._next_callbacks: list[CallbackType] = []
 
+    @property
+    def _prevent_message_types_stack(self) -> list[set[type[Message]]]:
+        """A stack that manages prevented messages.
+
+        Returns:
+            A list of sets of Message Types.
+        """
```

Comment:
```suggestion
        """The stack that manages prevented messages."""
```

https://github.com/Textualize/textual/wiki/Documentation-standards#property